src/olympia/reviewers/templates/reviewers/dashboard.html

```diff
@@ -12,7 +12,7 @@
         <h3>{{ section_name }}</h3>
         <ul>
           {% for text, href in sections[section_name] %}
-            <li><a href="{{ href }}">{{ text }}</a></li>
+            <li><a href="{{ href }}" {% if href.startswith(('https://', 'http://')) %}target="_blank" rel="noopener noreferrer"{% endif %}>{{ text }}</a></li>
```

Comment:
I can't help but think there should be cleaner way, or something like an `is_external` filter, to determine the link is external. But meh, I suppose this works.

Reply:
Eh actually I was lazy and didn't look for it but we have one. I'll try to use it.

Reply:
Meh, we don't expose it as a template function.